src/escape.c

```diff
@@ -258,6 +251,12 @@ bool checkEscapeRef(Scope *sc, Expression *e, bool gag)
                 ::error(loc, format, s->toChars());
             result = true;
         }
+        void error(Loc loc, const char *format, Expression *e)
```

Comment:
I know someone said that we should not do both refactoring and regression fix in a PR.